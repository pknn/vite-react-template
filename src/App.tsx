const App = () => <div>Vite React App</div>

export default App
