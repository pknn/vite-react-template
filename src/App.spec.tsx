import { shallow } from 'enzyme'
import { describe, it, expect } from 'vitest'
import App from './App'

const getShallowMounted = () => shallow(<App />)

describe('<App />', () => {
  it('should render correctly', () => {
    const component = getShallowMounted()

    expect(component).toMatchSnapshot()
    expect(component.text()).toBe('Vite React App')
  })
})
