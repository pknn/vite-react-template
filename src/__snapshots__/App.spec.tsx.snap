// Vitest Snapshot v1

exports[`<App /> > should render correctly 1`] = `ShallowWrapper {}`;
